x-pack/plugins/apm/public/components/routing/templates/apm_service_template/index.tsx

```diff
@@ -127,8 +125,9 @@ function TemplateWithContext({
       }}
     >
       <SearchBar {...searchBarOptions} />
-
-      {children}
+      <ServiceAnomalyTimeseriesContextProvider>
```

Comment:
Moving ServiceAnomalyTimeseriesContextProvider to here here so that it can get updates from `useEnvironmentsContext/ApmMainTemplate`